### ISSUE TITLE
Keyboard shortcuts and navigation

### ISSUE THREAD

**Issue by Cankyre** [franciscoBSalgueiro/en-croissant#251]
### Describe the feature

The keyboard navigation, especially when navigating PGN files, is a bit lacking. Here is a list of keyboard mappings that I'd like to see in En-croissant, including those from #11 . A lot of these can also be included in the menu bar.

**Tabs**
- [x] New tab (Ctrl+t)
- [x] Close tab (Ctrl+w)
- [x] Cycle tabs (Ctrl+Tab)
- [x] Cycle tabs backwards (Ctrl+Shift+Tab)  

**Files**
- [x] Open file (Ctrl+o)
- [x] Save file (Ctrl+s)
- [ ] Save as (Ctrl+shift+s)

**Edit**
- [ ] Comment position (d)
- [ ] Annotate position (g)
- [ ] Toggle annotations (1 to 6, "??" to "!!")
- [ ] Replace move (Shift+Move)
- [ ] Play best computer move (Space)
- [ ] Play first explorer move (Shift+Space)
- [x] Clear shapes (Ctrl+L)

**View**
- [x] Flip board (f)
- [x] Toggle eval bar and best moves arrows (z)
- [ ] Go to engines (e)
- [ ] Go to explorer (x)
- [ ] Toggle fullscreen (Ctrl+f)

**Navigation**
- [x] Next/previous move (right/left)
- [ ] Start of variation / End of variation (up/down)
- [x] Start of game / End of game (Shift+up/Shift+down)
- [ ] Next branch / Previous branch (Shift+right/Shift+left)

**Engine**
- [ ] Toggle all engines (Ctrl+e)
- [ ] Toggle specific engine (Ctrl+NUM)
- [ ] Generate game report (Ctrl+r)

**Play**
- [ ] Player vs. Engine (F9)
- [ ] Player vs. Player (F10)
- [ ] Engine vs. Engine (F11)

**Help**
- [ ] Show keybinds (?)
